### PR TITLE
Reorganized chef-backend.rb settings and add etcd.ionice settings (released in Chef Backend 2.2)

### PR DESCRIPTION
**Pull request by btm**
Move chef-backend.rb settings to the common place
    
    Having the Chef Backend settings deep under Setup -> Chef Infra Server
    -> Install High Availability, but not anywhere under Managing Chef Infra
    Server means I can never find them.
    
    All of the -ctl commands for Infra Server are under 'Managing Chef Infra
    Server' as is the chef-server.rb settings. This moves the the settings
    to the same location and links back. These rarely need to be modified,
    so they really should not be all spelled out at length on that page.
    
    There's a slight reordering of the menu to group the server, backend,
    and expander options in it.

--

    Add etcd.ionice configurations
    
    As of 2.1.13 (to be released in 2.2) you can configure the ionice values
    for etcd.

### CODE REVIEW

File: themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+The `chef-backend.rb` file is generated using
+`chef-backend-ctl gen-sample-backend-config` and controls most of the
+various feature and configuration flags going into a Chef HA backend
+node. A number of these options control the reliability, stability and
+uptime of the backend PostgreSQL databases, the elastic search index,
```
Comment (phiggins) [inline themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md]:
```suggestion
uptime of the backend PostgreSQL databases, the Elasticsearch index,
```

"Elasticsearch" is the name of the product, so this PR should probably have a `s/elastic search/Elasticsearch/g` run on it.

**Review by stevendanna — APPROVED** [review]
Technically the writeup looks great.  Thanks for working on this

File: content/config_rb_backend.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+-   `leaderl.health_check.interval_seconds` How frequently, in seconds,
+    to poll the service for health status. We recommend setting this to
+    at least 5 times the value of `leaderl.leader_ttl_seconds`. 5 by
+    default (2 by default before version 1.2)
+-   `leaderl.health_check.max_bytes_behind_leader` Limit on maximum
+    different between elected leader and current node in bytes.
+    `52428800` (50MB) by default.
+-   `leaderl.health_check.max_elasticsearch_failures` Number of Elastic
+    Search API failures allowed before health check fails. 5 by default.
```
Comment (phiggins) [inline content/config_rb_backend.md]:
```suggestion
-   `leaderl.health_check.max_elasticsearch_failures` Number of
    Elasticsearch API failures allowed before health check fails. 5 by default.
```

**Review by phiggins — APPROVED** [review]

File: themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+-   `fqdn` Host name of this node.
+-   `hide_sensitive` Set to `false` if you wish to print deltas of
+    sensitive files and templates during `chef-backend-ctl reconfigure`
+    runs. `true` by default.
+-   `ip_version` Set to either `'ipv4'` or `'ipv6'`. `'ipv4'` by
+    default.
+-   `publish_address` Externally resolvable IP address of this back-end
+    node.
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md]: Are these settings that the user could or should change?
I'm thinking prefacing this list with something like:
`Verify that the following settings are correctly configured:`
or
`Configure the following settings:`

Comment (btm) [reply themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md]: I think it is more like "these are the only settings you should change when setting up without guidance" - @sean-horn ?

File: content/config_rb_backend.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+    replication. `samehost` and `samenet` are special symbols to allow
+    connections from the this node's IP address and its subnet. You may
+    also use `all` to match any IP address. You may specify a hostname
+    or IP address in CIDR format (`172.20.143.89/32` for a single host,
+    or `172.20.143.0/24` for a small network. See
+    <https://www.postgresql.org/docs/9.5/static/auth-pg-hba-conf.html>
+    for alternative formats. `["samehost", "samenet"]` by default.
+-   `postgresql.replication_user` Username used by postgres streaming
+    replicator when accessing this node. `'replicator'` by default.
+-   `postgresql.username` `'chef_pgsql'` by default.
```
Comment (IanMadd) [inline content/config_rb_backend.md]: Is the `postgresql.username` the database administrator username? Something like:

`postgresql.username` Database administrator username, `'chef_pgsql'` by default.

**Comment by IanMadd** [comment]
Edited pages:
https://deploy-preview-2626--chef-web-docs.netlify.app/config_rb_backend/
https://deploy-preview-2626--chef-web-docs.netlify.app/install_server_ha/#chef-backendrb-options

File: content/config_rb_backend.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+Updating these settings may be advisable if you are seeing frequent
+failover events as a result of spurious etcd connection timeouts. The
+current defaults assume a high-latency environment, such those you might
+find if deploying Chef Backend to various cloud providers.
+
+-   `etcd.heartbeat_interval 500` ETCD_HEARTBEAT_INTERVAL in
+    milliseconds. This is the frequency at which the leader will send
+    heartbeats to followers. Etcd's documentation recommends that this
+    is set roughly to the round-trip times between members. (The default
+    before 1.2 was 100)
```
Comment (IanMadd) [inline content/config_rb_backend.md]: "(The default before 1.2 was 100)"
Does this refer to Chef Backend version 1.2?

**Review by IanMadd — APPROVED** [review]

**Pull request merged**